code/drasil-code/Language/Drasil/CodeSpec.hs

```diff
@@ -110,14 +110,14 @@ codeSpec SI {_sys = sys
 -- medium hacks ---
 
 -- Converts a chunk with a defining relation to a QDefinition
-relToQD :: ExprRelat c => ChunkDB -> c -> QDefinition
-relToQD sm r = convertRel sm (relat r)
+--relToQD :: ExprRelat c => ChunkDB -> c -> QDefinition
+--relToQD sm r = convertRel sm (relat r)
 
 -- Converts an Expr representing a definition (i.e. an equality where the left 
 -- side is just a variable) to a QDefinition.
-convertRel :: ChunkDB -> Expr -> QDefinition
-convertRel sm (EqBinaryOp Eq (C x) r) = ec (symbResolve sm x) r
-convertRel _ _ = error "Conversion failed"
+--convertRel :: ChunkDB -> Expr -> QDefinition
+--convertRel sm (EqBinaryOp Eq (C x) r) = ec (symbResolve sm x) r
+--convertRel _ _ = error "Conversion failed"
```

Comment:
Ah, just go ahead and remove relToQD and convertRel completely, no need to just comment them out. We can dig them up from git history if we want them back later.

Reply:
okay